tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -481,7 +472,7 @@ const loadNextInfiniteScrollPage = async () => {
 };
 
 const getErrorLog = async () => {
-  await errorLog().waitForDisplayed();
+  await (await $(`error-log`)).waitForDisplayed();
```

Comment:
Using regular single quotes
```suggestion
  await (await $('error-log')).waitForDisplayed();
```